nano/core_test/node.cpp

```diff
@@ -1682,26 +1682,40 @@ TEST (node, bootstrap_confirm_frontiers)
 }
 
 // Test that if we create a block that isn't confirmed, we sync.
-TEST (node, DISABLED_unconfirmed_send)
+TEST (node, unconfirmed_send)
```

Comment:
What is the intention of this test? What is it trying to test?
It is called unconfirmed send but I do not see anything unconfirmed in the test...
Which block is unconfirmed?